crates/bdk/src/wallet/coin_selection.rs

```diff
@@ -92,7 +92,10 @@
 //!     .unwrap();
 //! let psbt = {
 //!     let mut builder = wallet.build_tx().coin_selection(AlwaysSpendEverything);
-//!     builder.add_recipient(to_address.script_pubkey(), 50_000);
+//!     builder.add_recipient(
+//!         to_address.script_pubkey(),
+//!         bitcoin::Amount::from_sat(50_000),
+//!     );
```

Comment:
Isn't this covered in the

```rust
//! # use bitcoin::*;
```

on line 28?

```suggestion
//!     builder.add_recipient(
//!         to_address.script_pubkey(),
//!         Amount::from_sat(50_000),
//!     );
```